package.json

```diff
@@ -165,8 +165,8 @@
 						"highContrast": "#f0b800"
 					}
 				},
-				"cursorless.colors.purple": {
-					"description": "Color to use for purple symbols",
+				"cursorless.colors.pink": {
```

Comment:
this will break settings for people who have a custom purple.  was thinking of just pinging on slack before releasing?